arkouda/message.py

```diff
@@ -103,6 +102,19 @@ def _build_strings_param(key: str, val) -> ParameterObject:
         name = val.name if val.name else ""
         return ParameterObject(key, ObjectType.STRINGS, "str", name)
 
+    @staticmethod
+    def _is_supported_value(val):
+        import builtins
+        import numpy as np
+
+        return (
+            isinstance(val, str)
+            or isinstance(val, np.str_)
+            or isSupportedNumber(val)
+            or isinstance(val, builtins.bool)
+            or isinstance(val, np.bool_)
+        )
```

Comment:
i think we could make this a shorter by combining the `isinstance`s
```python
return isSupportedNumber(val) or isinstance(val, (str, np.str_, builtins.bool, np.bool_))
```

Reply:
Oh yeah good catch. This is updated.